### ISSUE TITLE
Accessibility: color contrast on link does not meet min requirements

### ISSUE THREAD

**Issue by sarahmhgov** [18F/tock#1565]
The link on the tock submit view to view "billable hours" is not sufficient.

### Steps to Reproduce:

Navigate to https://tock.18f.gov/reporting_period/#
Scroll down to 'add item', "billable hours" link

Expected: Blue link should meet color contrast
Actual: Blue link does not meet color contrast requirements. 

WCAG Success Criteria (A): 1.4.3 Contrast (Minimum)

### Remediation guidance:

Change the background to a lighter color to increase the contrast.

**Comment by melchoyce** [comment]
I'm getting a 404 on that link—how would I navigate to that page from the tock homepage?

**Comment by katefisher808** [comment]
@sarahmhgov is this ticket referencing the blue background in the screenshot attached? 

![Image](https://user-images.githubusercontent.com/125484578/226634865-d44f2310-bad4-4250-98a2-414830406b6a.png)



cc @melchoyce

**Comment by jasnakai** [comment]
PR #1580 created for resolution of issue.
